python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -180,16 +179,49 @@
     'handlers': {
         'console': {
             'class': 'logging.StreamHandler',
+            'level': 'DEBUG',
             'formatter': 'verbose',
         },
+        'logfile': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'formatter': 'verbose',
+            'filename': os.path.join(BASE_DIR, 'cac-tripplanner-app.log'),
+        },
     },
     'loggers': {
         'django': {
-            'handlers': ['console'],
-            'level': 'INFO',
-        }
+            'handlers': ['console', 'logfile'],
+            'level': 'DEBUG',
```

Comment:
Is all this DEBUG-level logging necessary/useful?  Seems like it would be noisier than we'd want.

Reply:
Probably, yeah. I was trying to see if I could get any useful info for debugging an issue, but it is pretty noisy (and didn't help me anyway). I'll back it off to `INFO`.